Rakefile

```diff
@@ -51,6 +51,7 @@ end
   ["msi:build",        "Build MSI package (alias for msi:dockerbuild)"],
   ["msi:selfbuild",    "Build MSI package without using Docker"],
   ["msi:dockerbuild",  "Build MSI package by Docker"],
+  ["dmg:selfbuild",    "Build macOS package without using Docker"],
```

Comment:
If we use `dmg:selfbuild` instead of `macos:selfbuild`, directory name should be also `dmg` to unify naming rule.

Reply:
Sure. Will do.